grouper/models.py

```diff
@@ -253,6 +253,10 @@ def name(self):
     def type(self):
         return "User"
 
+    @property
+    def is_enabled(self):
+        return self.enabled
```

Comment:
I thought it wasn't used anywhere outside of the User class, but on closer inspection, both the graph and the fe handlers read it directly, so it makes sense to keep that consistent. Fixing!